omod/src/main/webapp/resources/htmlforms/HMIS-HIV-003-HivCareArtCard-ClinicalAssessmentPage.xml

```diff
@@ -913,581 +1133,1838 @@
+                                                                 class="horizontal"/>
+                                                        </div>
+                                                        <div class="col-2">
+                                                            <obs labelText="No. of Pills" conceptId="160856"/>
+                                                        </div>
+                                                        <div class="col-2 duration">
+                                                            <obs conceptId="159368" labelText="No. of Days"/>
+                                                        </div>
+
+                                                        <div class="col-2">
+                                                            <obs id="br" conceptId="1190" labelText="Start Date"/>
+                                                        </div>
+                                                        <div class="col-2">
+                                                            <obs id="br" conceptId="1191" labelText="Stop Date"/>
+                                                        </div>
+                                                    </div>
+                                                </obsgroup>
                                             </div>
                                         </div>
-                                    </div>
-                                    <div class="card">
-                                        <div class="card-header">Advanced Disease Status and WHO Clinical Stage</div>
-                                        <div class="card-body">
-                                            <div class="row">
-                                                <div class="col-md-2">
-                                                    <label>Advanced Disease Status</label>
-                                                </div>
-                                                <div class="col-md-3">
-                                                    <obs conceptId="165272" class="long-value-select" answerConceptIds="165309, 165266, 165267, 165268, 165269, 165270, 165271"
-                                                         answerLabels="1 - No Adv. Disease, 2 - Suspected Adv. Disease, 3a - Confirmed Adv. Disease CD4 &lt; 200, 3b - Confirmed Adv. Disease - Pos CrAg,
-                                                         3c - Confirmed Adv. Disease - Pos TB, 3d - Confirmed Adv. Disease - WHO Stage 3/4, 3e - Confirmed Adv. Disease - Child &lt; 5 Yrs"/>
-                                                </div>
-                                                <div class="col-md-1"></div>
-                                                <div class="col-md-2">
-                                                <label>WHO Clinical stages</label>
+                                        <div class="card" style="margin-top:10px">
```

Comment:
@slubwama remove margin-top, as a rule no margins or paddings are added throughout the forms to keep them consistent and responsive

Reply:
done